src/main/java/ch/njol/skript/structures/StructFunction.java

```diff
@@ -35,49 +35,66 @@
 import org.skriptlang.skript.lang.structure.Structure;
 
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.regex.MatchResult;
 
 @Name("Function")
 @Description({
 	"Functions are structures that can be executed with arguments/parameters to run code.",
-	"They can also return a value to the trigger that is executing the function."
+	"They can also return a value to the trigger that is executing the function.",
+	"Note that local functions always come before global functions, in parsing and execution"
 })
 @Examples({
 	"function sayMessage(message: text):",
 	"\tbroadcast {_message} # our message argument is available in '{_message}'",
-	"function giveApple(amount: number) :: item:",
-	"\treturn {_amount} of apple"
+	"local function giveApple(amount: number) :: item:",
+	"\treturn {_amount} of apple",
 })
-@Since("2.2")
+@Since("2.2, INSERT VERSION (local functions)")
 public class StructFunction extends Structure {
 
-	public static final Priority PRIORITY = new Priority(400);
+	public static final Priority GLOBAL_PRIORITY = new Priority(400);
+	public static final Priority LOCAL_PRIORITY = new Priority(450);
```

Comment:
Wait, are local functions supposed to be parsed *first*? If so, the priorities here are mixed up.

Reply:
Global functions are parsed first in declaration but local functions are called first. The parsing order doesn't really affect anything